### ISSUE TITLE
Input hints

### ISSUE THREAD

**Issue by halimahLDN** [Greater-London-Authority/ldn-viz-tools#1313]
In looking into refactoring the `LayerControlGroup` and `LayerControl` components to allow custom overlays, I've discovered that the way inputs use tooltips does not follow the accessibility pattern correctly.

See [MDN ARIA: tooltip role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/tooltip_role#:~:text=The%20tooltip%20is%20not%20the,the%20tool%20or%20control%20is.)

> The tooltip is not the appropriate role for the more information "i" icon, ⓘ. A tooltip is directly associated with the owning element. The ⓘ isn't 'described by' detailed information; the tool or control is.

As such, tooltips should be triggered by hovering on/focusing the input or its label. Otherwise, the more information button should probably default to opening a popover or modal.

It should also always have a label of some kind - if not visible, an `aria-label` attribute should be added.
